whole-body-controller-core/robot-arm-one/src/main/java/us/ihmc/robotArmOne/SevenDoFArmParameters.java

```diff
@@ -148,7 +154,7 @@ public Graphics3DObject getChildLinkGraphics()
    public static final EnumMap<SevenDoFArmJointEnum, Double> jointChildLinkMasses = new EnumMap<>(SevenDoFArmJointEnum.class);
    public static final EnumMap<SevenDoFArmJointEnum, Vector3D> jointChildLinkCoMs = new EnumMap<>(SevenDoFArmJointEnum.class);
    public static final EnumMap<SevenDoFArmJointEnum, Matrix3D> jointChildLinkInertias = new EnumMap<>(SevenDoFArmJointEnum.class);
-   public static final EnumMap<SevenDoFArmJointEnum, Graphics3DObject> jointChildLinkGraphics = new EnumMap<>(SevenDoFArmJointEnum.class);
+   public static final EnumMap<SevenDoFArmJointEnum, RigidBodyDefinition> jointChildRigidBody = new EnumMap<>(SevenDoFArmJointEnum.class);
```

Comment:
That's a little weird to have a `RigidBodyDefinition` map with this enum. I'd propose changing that into `VisualDefinition` map which corresponds to what it used to be.

Reply:
Done.